packages/web/src/components/edit-track/EditTrackForm.tsx

```diff
@@ -187,7 +187,7 @@ const TrackEditForm = (
     // we will listen to all changes from the user, rather than just a new image from
     // the backend.
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [updatedArtwork?.url, setArtworkValue])
+  }, [updatedArtwork, setArtworkValue])
```

Comment:
Given the comment above, just wondering why this got changed? Seems like it will cause a lot more re-evaluations of the effect.

Reply:
ah yeah it's to disambiguate types. added a helper. will clean this up too